versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/DisplayPluginUpdatesMojo.java

```diff
@@ -408,11 +418,11 @@ public void execute()
         Map<String, String> parentReportPlugins = new HashMap<>();
 
         Set<Plugin> plugins = getProjectPlugins( superPomPluginManagement, parentPlugins, parentBuildPlugins,
-                                                 parentReportPlugins, pluginsWithVersionsSpecified );
+                parentReportPlugins, pluginsWithVersionsSpecified );
 
         List<String> pluginUpdates = new ArrayList<>();
         List<String> pluginLockdowns = new ArrayList<>();
-        ArtifactVersion curMavenVersion = runtimeInformation.getApplicationVersion();
+        ArtifactVersion curMavenVersion = new DefaultArtifactVersion( runtimeInformation.getMavenVersion() );
         ArtifactVersion specMavenVersion = MinimalMavenBuildVersionFinder.find( getProject(), "2.0", getLog() );
```

Comment:
To investigate if code is needed for Maven 3.x

Reply:
Changed from `2.0` to `3.2.5` as that is the minimum version. This is still necessary if the user is running a older than current version of Maven, but > 3.2.5